### PR TITLE
chore(Upgrades): several upgrades

### PR DESCRIPTION
**Pull request by eyenx**
## infra-apps
* velero
* rbacManager
* argocd
* ingress-nginx
## misc-apps
* downscaler
* metallb
## logging-apps
* filebeat
* fluentd
## tracing-apps
* fix Version of jaeger-operator
## security-apps
* fix repoURL of gatekeeper

### CODE REVIEW

**Comment by tongpu** [comment]
Are there any relevant upgrade notices that we should put in a changelog?

File: charts/logging-apps/values.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ fluentd:
   # fluentd.chart -- Chart
   chart: "fluentd"
   # fluentd.targetRevision -- [fluentd Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/fluentd) version
-  targetRevision: "3.0.*"
+  targetRevision: "3.2.*"
```
Comment (eyenx) [inline charts/logging-apps/values.yaml:56]: Introduction of `bitnami/common` (library dependency) https://github.com/bitnami/charts/tree/master/bitnami/fluentd#to-310

File: charts/infra-apps/values.yaml [hunk 2]
```diff
@@ -53,7 +53,7 @@ ingressNginx:
   # ingressNginx.chart -- Chart
   chart: "ingress-nginx"
   # ingressNginx.targetRevision -- [ingress-nginx Helm chart](https://github.com/kubernetes/ingress-nginx/tree/master/charts/ingress-nginx) version
-  targetRevision: '3.7.*'
+  targetRevision: '3.17.*'
```
Comment (eyenx) [inline charts/infra-apps/values.yaml:56]: Depracation of helm2 support + update to v0.42.0 ingress-nginx 

* https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/CHANGELOG.md#3120
* https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/CHANGELOG.md#3170

File: charts/misc-apps/values.yaml [hunk 3]
```diff
@@ -13,7 +13,7 @@ downscaler:
   # downscaler.chart -- Chart
   chart: "kube-downscaler"
   # downscaler.targetRevision -- [kube-downscaler Helm chart](https://github.com/helm/charts/tree/master/incubator/kube-downscaler) version
-  targetRevision: "0.5.*"
+  targetRevision: "0.6.*"
```
Comment (eyenx) [inline charts/misc-apps/values.yaml:16]: Version `20.10.0` of kubedownscaler image https://github.com/helm/charts/commit/fc37da771f6c3d405b0e2b3d39f3a0e5a1732c67#diff-7c6021564b8e31f7e48abc401a2a48fb56f5900e63e3b29438dce4219d9d89e2

File: charts/misc-apps/values.yaml [hunk 4]
```diff
@@ -73,7 +73,7 @@ metallb:
   # metallb.chart -- Chart
   chart: "metallb"
   # metallb.targetRevision -- [metallb Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/metallb)
-  targetRevision: "1.0.*"
+  targetRevision: "2.0.*"
```
Comment (eyenx) [inline charts/misc-apps/values.yaml:76]: Upgrade Notice:

https://github.com/bitnami/charts/tree/master/bitnami/metallb#to-200

File: charts/tracing-apps/values.yaml [hunk 5]
```diff
@@ -13,7 +13,7 @@ jaegerOperator:
   # jaegerOperator.chart -- Chart
   chart: "jaeger-operator"
   # jaegerOperator.targetRevision -- [jaeger-operator Helm chart](https://github.com/jaegertracing/helm-charts/tree/master/charts/jaeger-operator)
-  targetRevision: "1.20.*"
+  targetRevision: "2.18.*"
```
Comment (eyenx) [inline charts/tracing-apps/values.yaml:16]: wrong chart version was used!

**Review by tongpu — APPROVED** [review]
LGTM

**Comment by eyenx** [comment]
`Chart.yaml` version bump missing on 

* logging-apps
* misc-apps
* security-apps

**Comment by eyenx** [comment]
Okay now it looks better.

**Pull request merged**